commands/Moderation/nickname.js

```diff
@@ -0,0 +1,29 @@
+exports.run = async (client, msg, [user, name]) => {
```

Comment:
You should put name as optional, defaulting to `""`, so the bot can clear nicknames.

Reply:
```[user, name = ""]```